presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/QueryBuilder.java

```diff
@@ -278,4 +293,23 @@ private static void bindValue(Object value, Type type, List<TypeAndValue> accumu
         checkArgument(isAcceptedType(type), "Can't handle type: %s", type);
         accumulator.add(new TypeAndValue(type, value));
     }
+
+    private static long toLong(Object value)
+    {
+        if (value instanceof Long) {
```

Comment:
Actually, I'm not sure this is needed. Are the values not always the correct type?